packages/fxa-payments-server/src/components/Header/index.tsx

```diff
@@ -1,30 +1,41 @@
 import React from 'react';
-
-import './index.scss';
 import { Profile } from '../../store/types';
+import firefoxLogo from '../../images/firefox-logo-combined.svg';
+import { useLocalization } from '@fluent/react';
 
-type HeaderProps = {
+export type HeaderProps = {
   profile?: Profile;
   className?: string;
 };
 
 export const Header = ({ profile, className = 'default' }: HeaderProps) => {
+  const { l10n } = useLocalization();
   let profileSection = null;
+
   if (profile) {
     const { avatar, displayName, email } = profile;
     profileSection = (
       <img
-        className="avatar hoisted"
+        className="rounded-full w-8 h-8"
         src={avatar}
         data-testid="avatar"
         alt={displayName || email}
+        title={displayName || email}
       />
     );
   }
 
   return (
-    <header className={`${className} fxa-settings-header`} role="banner">
-      <div className="fxa-logo" data-testid="branding" title="firefox"></div>
+    <header
+      className={`${className} flex justify-between items-center bg-white shadow h-16 fixed left-0 top-0 my-0 mx-auto px-7 py-4 w-full z-10 tablet:h-20`}
+      role="banner"
+    >
+      <div data-testid="branding">
+        <img
```

Comment:
This might need a width of 120px? It looks smaller than before.

Reply:
I pulled the image out of the background. It's width is 117px. Added arbitrary value for now as it will be updated in FXA-6233.